### PR TITLE
Upload coverage to codecov action

### PR DESCRIPTION
**Pull request by g4s8**
#44 - upload coverage to codecov action

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#45](https://github.com/artipie/go-adapter/issues/45) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/92b7f630-0601-46b7-be0f-e8714aca6c3b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
 
 [![Build Status](https://img.shields.io/travis/artipie/go-adapter/master.svg)](https://travis-ci.org/artipie/go-adapter)
-[![Javadoc](http://www.javadoc.io/badge/com.artipie/goproxy.svg)](http://www.javadoc.io/doc/com.artipie/goproxy)
+[![Codecov branch](https://img.shields.io/codecov/c/gh/artipie/go-adapter/master)](https://codecov.io/gh/artipie/go-adapter/branch/master)
+[![PDD status](http://www.0pdd.com/svg?name=artipie/go-adapter)](http://www.0pdd.com/p?name=artipie/go-adapter)
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.artipie/go-adapter.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/go-adapter)
+[![Javadoc](http://www.javadoc.io/badge/com.artipie/go-adapter.svg)](http://www.javadoc.io/doc/com.artipie/go-adapter)
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/artipie/goproxy/blob/master/LICENSE.txt)
 [![Hits-of-Code](https://hitsofcode.com/github/artipie/go-adapter)](https://hitsofcode.com/view/github/artipie/go-adapter)
```
Comment (Vatavuk) [inline README.md:14]: @g4s8 I would group hits of code with code coverage and build just to be in the same context

**Comment by g4s8** [comment]
@Vatavuk thanks, fixed

**Comment by 0crat** [comment]
Job was finished in 8 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks!

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**